monailabel/monailabel_endoscopy_cvat_tooltracking.ipynb

```diff
@@ -41,6 +41,14 @@
     "- **Install MONAI Label** weekly preview release: "
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup environment"
```

Comment:
Hi @tangy5 , can you put the 

```
Prerequisites
Install MONAI Label weekly preview release:
```
under the markdown cell that has "## Setup environment"? 
The reason behind is that the text line is part of environment setup, and can be removed when we do not need to set up the environment in a container.

Reply:
Sure